OHM-Trade-Agent-v1/app/services/qualified_alert_outbox.py

```
@@ -0,0 +1,305 @@
+    chat_id: str,
+) -> str:
+    claim = _claim(trade_id)
+    if claim is None:
+        return "BUSY_OR_MISSING"
+    lease_token, row = claim
+
+    try:
+        plan = EntryExitPlan(**dict(row.get("plan") or {}))
+        if str(row.get("trade_id") or "") != trade_id:
+            raise ValueError("qualified alert outbox trade_id mismatch")
+    except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)